scripts/prepublish

```diff
@@ -5,7 +5,8 @@ README="$(cat README.md)"
 README="${README//"$PREVIOUS_VERSION/ramda.min.js"/"$VERSION/ramda.min.js"}"
 README="${README//"${PREVIOUS_VERSION%.*}/ramda.min.js"/"${VERSION%.*}/ramda.min.js"}"
 echo "$README" >README.md
-git add README.md
+npm run bookmarklet
+git add README.md BOOKMARKLET.md
```

Comment:
Let's devote one paragraph to each file:

    paragraph for the creation
    and addition of README.md

    paragraph for the creation
    and addition of BOOKMARKLET.md

    paragraph for the creation
    and addition of dist/ramda{,.min}.js

In other words, leave the existing two paragraphs unchanged and add a new paragraph:

```bash
npm run bookmarklet
git add BOOKMARKLET.md
```

I'm sorry that I didn't make this clear in my previous comment.